.ci/gcb-generate-diffs-new.yml

```diff
@@ -129,7 +129,7 @@ steps:
       args:
           - 'base'
           - 'terraform-google-conversion'
-          - 'ga'
```

Comment:
we should temporarily switch this to GA so that the tests can run (even though they'll fail)